damus/Views/Video/DamusVideoPlayer.swift

```diff
@@ -93,6 +96,27 @@ struct DamusVideoPlayer: View {
             }
         }
     }
+    
+    private var live_indicator: some View {
+        ZStack {
+            VStack {
+                HStack {
+                    Capsule()
+                        .fill(Color.black.opacity(0.5))
+                        .frame(width: 80, height: 30)
```

Comment:
You might not want to specify the frame explicitly since there's a localized string inside it. Sometimes German is 2.5X the length of the English.